### DISCUSSION TITLE
My eQSLs all show as not sent

### DISCUSSION THREAD

**Discussion by davejhahn** [magicbug/Cloudlog#807] [Q&A]
I use the import from eQSL and everything incoming looks fine (e.g. shows the green up arrow as received) but the first arrow for sent is red (not sent) for all my QSOs.

I automatically send through JTAlert, everything is definitely in the outbox in eQSL.

I assumed that the Upload QSOs tab would take care of this, but I always get a Internal Server Error when running.  There are ~950 QSOs in there.  

Not really needing to send them to eQSL, because they are already there, I then tried using the Tools tab and "Mark all QSOs as Sent to eQSL", it indicates "All QSOs Marked as Uploaded", but when I go back to my Log, they are all still red.

Maybe I'm missing something, but I believe I'm using the correct features--one errors out the other doesn't seem to do anything.

Any ideas?

**Comment by magicbug** [comment]
Internal Server Errors coming from the server scripts timing out before its completing depends on hosting company if this can be increased.

Currently we don't mark QSO is sent based on RX although we could do that, usually I take it that imported ADIF will have that marked for us.

Pete.

**Reply by davejhahn** [reply]
On the timeout, not sure if you mean eQSL's side or CloudLog.  I'm using cloudlog.co.uk   Can't tell which side is actually timing out.

I guess I'm not sure what flow to follow then...I like CloudLog because it is actually a modern UI that has useful capabilities unlike most if not all of the primary logging sites (eQSL, QRZ, LoTW, HRDLog, HamQTH, ClubLog).  I basically aggregate everything into CloudLog.

What I do is use JTAlert's ADIF log to import into CloudLog.  Even though JTAlert sends via web service to QRZ, HRDLog, HamQTH, eQSL, and ClubLog, which it does fine, it doesn't mark eQSL in the ADIF log, EQSL_QSL_RCVD and EQSL_QSL_SENT are all set to "R" (requested), so you  mentioned you assume that it is marked, but in my case it is not.

So what I do then, is, for the two that CloudLog tracks, eQSL and LoTW, get those synced with the following: 

For LoTW, I'll sign the same JTAlert ADIF file and upload to LoTW and when processed, use the LoTW Import in Cloud Log.   That updates 2 way statuses, sent/received.  Works beautifully. 

But for eQSL, although I can use "Import eQSL QSO Matches" in CloudLog, which updates the received fine, e.g. can see the status and the QSL image, but unlike LoTW, it doesn't reflect the Sent.

I thought that Upload QSOs might set the flag, but I mentioned that times out.   Looking further, I assumed that the "Mark All QSOs as Sent to eQSL" would do exactly what I'm trying to do, mark everything as sent (which is true, since JTAlert automatically sends everything).  It indicates "All eQSLs Marked as Uploads" when I run it, but it doesn't seem like it actually does anything at all, e.g. the statuses do not change at all.  What is this feature actually supposed to do?

I suppose I could update the ADIF file for the Sent status going forward, but I'm assuming for any of the existing ones it will just ignore it as a duplicate.

Lots of babbling, sorry, but I guess main question is what does eQSL Tools -> "Mark All QSOs as Sent to eQSL" do?
